charts/cla/templates/ingress.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "cla.fullname" . }}
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    nginx.ingress.kubernetes.io/limit-rps: "10"
+spec:
+  tls:
+  - hosts:
+      - {{ .Values.claAssistant.host | default "example.com" }}
+    secretName: {{ include "cla.fullname" . }}-tls
+  rules:
+  - host: {{ .Values.claAssistant.host | default "example.com" }}
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: cla-backend
```

Comment:
we shoud use here the same template we use in the service resource; also for the port.